pom.xml

```diff
@@ -319,21 +307,6 @@
   </pluginRepositories>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-site-plugin</artifactId>
-          <dependencies>
-            <dependency>
-              <groupId>com.github.stephenc.wagon</groupId>
-              <artifactId>wagon-gitsite</artifactId>
-              <version>0.5</version>
-            </dependency>
```

Comment:
https://github.com/stephenc/wagon-gitsite has not been updated since 2014, so this is a dead end.